docs/api/helpers/PlaneHelper.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<base href="../../" />
+		<script src="list.js"></script>
+		<script src="page.js"></script>
+		<link type="text/css" rel="stylesheet" href="page.css" />
+	</head>
+	<body>
+		[page:LineSegments] &rarr;
+
+		<h1>[name]</h1>
+
+		<div class="desc">
+			Helper object to visualize a [page:Plane].
+		</div>
+
+
+		<h2>Example</h2>
+
+		<code>
+		var plane = new THREE.Plane(new THREE.Vector3(1, 1, 0.2), 3);
+		helper = new THREE.PlaneHelper( plane, 1, 0xffff00 );
+		scene.add( helper );
+		</code>
+
+
+		<h2>Constructor</h2>
+
+
+		<h3>[name]( [page:Plane plane], [page:Float size], [page:Color hex] )</h3>
+		<div>
+		[page:Plane plane] --  the plane to visualize.<br />
+		[page:Float size] -- (optional) side length of plane helper. Default is 1.
+		[page:Color color] --  (optional) the color of the helper. Default is 0xffff00.<br /><br />
+
+		Creates a new wireframe representation of the passed plane.
+		</div>
+
+		<h2>Properties</h2>
+		<div>See the base [page:LineSegments] class for common properties.</div>
+
+		<h3>[property:Plane plane]</h3>
+		<div>The [page:Plane plane] being visualized.</div>
+
+		<h3>[property:Float size]</h3>
+		<div>The side lengths of plane helper.</div>
+
+
+		<h2>Methods</h2>
+		<div>See the base [page:LineSegments] class for common methods.</div>
+
+		<h3>[method:void updateMatrixWorld]( force )</h3>
+		<div>
+			Overridden to also update the helper size and location according to passed
+			[page:PlaneHelper.plane] and the [page:PlaneHelper.size] property.
```

Comment:
This could be clearer. I guess you mean that it overrides the base class method? 

Do you need to set it to true if the size parameter is changed? Or if the Plane being represented is moved?

Reply:
I'm trying to say that the `updateMatrixWorld` function has been overridden such that it will automatically update the helper whenever this function is called. I don't have a good enough overview to know in which scenarios you need to set `matrixWorldNeedsUpdate = true` for this to be called. If we need a note about that, I'd have to defer to someone else.

Reply:
OK, I don't really have strong opinion about this - methods like `updateMatrixWorld` tend to be used by more advanced users anyway and they can read the source code for info on how the method works. 

However I do think it can be worded more clearly  - something like

> This overrides the method in the base [page:LineSegments] class so that it also updates the [page:PlaneHelper.plane .plane] and [page:PlaneHelper.size .size] properties. 

Same goes for Box3Helper.updateMatrixWorld btw.

Reply:
OK, I don't really have strong opinion about this - methods like `updateMatrixWorld` tend to be used by more advanced users anyway and they can read the source code for info on how the method works. 

However I do think it can be worded more clearly  - something like

> This overrides the method in the base [page:LineSegments] class so that it also updates the [page:PlaneHelper.plane .plane] and [page:PlaneHelper.size .size] properties. 

Same goes for Box3Helper.updateMatrixWorld btw.

Reply:
Thanks for clearing that up. Is this ok? 
> This overrides the method in the base [page:Object3D] class so that it also updates the helper according to the [page:PlaneHelper.plane] and [page:PlaneHelper.size] properties.

Reply:
Yep, just please put `[page:PlaneHelper.size .size]` since we are referring to a property on the current page (the second .size will be the text for the link).